test/integration_test_labeled.py

```diff
@@ -233,6 +233,11 @@ def check_start_end_dates(row):
         )
 
     def test_all_older_features_have_24_months(self):
+        # If this test is failing, datasets with start dates two years before today
+        # (technically 2 years and 4 months, see relative delta below for cutoff) do not have
+        # features for the 24 month period following their start date.
+        # To address this, update the failing datasets by adding current features.
```

Comment:
Why not put something along these lines in the test failure message?
(I think it can probably be condensed a bit)